em_workflows/brt/flow.py

```diff
@@ -526,39 +484,39 @@ def gen_ng_metadata(fp_in: FilePath) -> Dict:
     return ng_asset
 
 
-with Flow(
-    "brt_flow",
-    executor=BRTConfig.SLURM_EXECUTOR,
-    state_handlers=[utils.notify_api_running],
-    terminal_state_handler=utils.custom_terminal_state_handler,
-    run_config=LocalRun(labels=[utils.get_environment()]),
-) as flow:
+# run_config=LocalRun(labels=[utils.get_environment()]),
+@flow(
+    name="Flow: BRT",
+    log_prints=True,
+    task_runner=BRTConfig.SLURM_EXECUTOR,
+    on_completion=utils.notify_api_completion,
+    # on_failure=utils.notify_api_completion,
```

Comment:
it seems strange they don't have a helper like `on_start` - (we could put the call into the API there)